installer/volcano-agent-development.yaml

```diff
@@ -127,6 +134,11 @@ spec:
       initContainers:
         - name: volcano-agent-init
           image: docker.io/volcanosh/vc-agent:latest
+          securityContext:
```

Comment:
Agent need more privilege, this need to be verified.

Reply:
Add more privilege: DAC_OVERRIDE,SETUID,SETGID,SETFCAP